### DISCUSSION TITLE
关于自己个人的测试版使用询问

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#3095] [提问] [answered]
我有电脑A和B，A有测试版，B没有，我将A的测试版文件复制到B，是否违反协议？
_记得问过来着但是忘了，麻烦回答一下_

**Comment by zkitefly** [comment]
我觉得不违反，因为没有公开传播

**Comment by WForst-Breeze** [comment]
你自己的电脑是可以的，因为要求是不允许传播给**他人**

**Reply by Deep-Dark-Forest** [reply]
那么……~~Pcl2知道我到底是自己的电脑还是别人的电脑吗~~

**Reply by WForst-Breeze** [reply]
这个就靠自觉喽

**Reply by iPodtouch6557** [reply]
~说回来这类功能在1980s就有了，一个内置病毒可以做到这一点~

**Reply by burningtnt** [reply]
~~检测鼠标移动习惯~~

**Reply by Deep-Dark-Forest** [reply]
> ~~检测鼠标移动习惯~~

~~好好好，开发者，请支付所有开发所得收入~~

**Reply by allMagicNB** [reply]
> > ~~检测鼠标移动习惯~~
> 
> ~~好好好，开发者，请支付所有开发所得收入~~

~~检测文档\Tencent Files\下的QQ号有没有进入QQ群~~

**Reply by iPodtouch6557** [reply]
不，是当检测到文件被复制时，病毒就会激活
基本上DOS都要先复制再说，纯命令行嘛

**Comment by allMagicNB** [comment, marked as answer]
不违反的，因为没有**公开发布**快照版
~~虽然B电脑不支持更新快照版版本，但是之后可更换识别码时换一下就行了~~
